Makefile

```diff
@@ -21,6 +21,23 @@ test:
 	bash scripts/test_no_wip_tasks_remaining.sh
 	npm run lint
 
+# Run playwright tests. show-report will run automatically if a test
+# fails.
+.PHONY: test-ui
+test-ui:
```

Comment:
Might need to add a step to install the browsers

`npx playwright install`

Reply:
Thanks! I have added a playwright-browsers makefile target dependency to all the playwright related targets.

https://github.com/barkbank/barkbank-pawtal/pull/101/commits/c398230195f4ef1026caa0e7bb1812582f463bbf